tests/Filesystem/FilesystemTest.php

```diff
@@ -271,11 +271,16 @@ public function testIsReadable()
     {
         file_put_contents($this->tempDir.'/foo.txt', 'foo');
         $files = new Filesystem();
-        @chmod($this->tempDir.'/foo.txt', 0000);
-        $this->assertFalse($files->isReadable($this->tempDir.'/foo.txt'));
-        $this->assertFalse($files->isReadable($this->tempDir.'/bar.txt'));
-        @chmod($this->tempDir.'/foo.txt', 0777);
-        $this->assertTrue($files->isReadable($this->tempDir.'/foo.txt'));
+        // chmod is noneffective on Windows
+        if (DIRECTORY_SEPARATOR === '\\') {
```

Comment:
Habits… Though, the helper is used nowhere in the tests, so I'll just keep it as so.